dev-libs/tree-sitter-haskell/Manifest

```diff
@@ -1 +1 @@
-DIST tree-sitter-haskell-0.19.9999_p20211114.tar.gz 834925 BLAKE2B d7cadc960ddb49efa6c439dfc13b674d398a4576fb85fd5ef12b4015049fd629722fe0d2788ab4c0052f1a3a6baa9fe5de925d86f7098495429190871340823b SHA512 db3f2ddd20195521a5a03d6ecf1305dc222343ba54ebdabf0cc68db680553908ce6d0ccba4fdcf9073e4f5187b2a6fae6364bda935167ca8d4e421500a86fdfa
```

Comment:
What's going on here? Is it similar to sam's comment on the python grammar?

Reply:
yes

// btw, github somewhy says you're commenting outdated revision:
![image](https://user-images.githubusercontent.com/268102/145309020-0c4cc080-87e7-49a8-943c-6bbdec43be0e.png)

Reply:
oh yeah sorry, this is gone in the latest revision